modules/web/js/ballerina/model/default-node-factory.js

```diff
@@ -568,6 +568,9 @@ class DefaultNodeFactory {
             node.setVariables(returnNode.getVariables());
         }
         node.getExpression().getName().setValue(functionDef.getName());
+         if (packageName && packageName !== 'Current Package' && packageName !== "builtin" ) {
```

Comment:
use single quotes instead of double quotes

Reply:
Fixed with 6dbe0dd89c23f52172acc6bc2be1a8a3d69fa440